jsconfig.json

```diff
@@ -0,0 +1,5 @@
+"compilerOptions": {
```

Comment:
@georgiana-b 

1. This file is invalid JSON
2. Why is it being committed to the codebase?